src/provider/store/Actions.ts

```diff
@@ -89,7 +90,13 @@ export class ToggleVisibility extends BaseAction<Action.TOGGLE_VISIBILITY> {
     }
 }
 
-export type RootAction = CreateNotification|RemoveNotifications|ClickNotification|ClickButton|ToggleVisibility;
+export class BlurCenter extends BaseAction<Action.BLUR_CENTER> {
```

Comment:
Instead of creating another action maybe expand the `ToggleVisibility` action by adding a some sort of "source" property.
```ts
type ToggleSource = 'api' | 'tray' | 'blur'
class ToggleVisibility...{
	source: ToggleSource;
	constructor(reason: ToggleSource, visible?: boolean)
	...
```

Then in `index.ts`
```ts
else if (action.type === Action.TOGGLE_VISIBILITY){
	if(action.source === 'blur'){
		this._toggleBlockTimer.start();
	}
}
```